source/discretization.h

```diff
@@ -113,7 +121,8 @@ DECLARE_ENUM(ryujin::Boundary,
                   {ryujin::Boundary::slip, "slip"},
                   {ryujin::Boundary::no_slip, "no_slip"},
                   {ryujin::Boundary::dirichlet, "dirichlet"},
-                  {ryujin::Boundary::dynamic, "dynamic"}));
+                  {ryujin::Boundary::dynamic, "dynamic"},
+                  {ryujin::Boundary::discharge_only, "discharge only"}));
```

Comment:
I am not a huge fan of the name `discharge_only`. Can we find something more general? `dirichlet_momentum`, `dirichlet_momentum_only`?

Reply:
`dirichlet_momentum` is fine with me.

Reply:
Pushed new commit to fix this.